common/utils.js

```diff
@@ -785,6 +785,40 @@ function validateDestinationChain(chains, destinationChain) {
     validateChain(chains, destinationChain);
 }
 
+async function estimateITSFee(chain, destinationChain, env, eventType, gasValue, _axelar) {
+    if (env === 'devnet-amplifier') {
+        printInfo('Using default ITS fee: 0');
+        return 0;
+    }
+
+    if (isValidNumber(gasValue)) {
+        return scaleGasValue(chain, gasValue);
+    }
+
+    const url = `${_axelar?.axelarscanApi}/gmp/estimateITSFee`;
+
+    const payload = {
+        sourceChain: chain.axelarId,
+        destinationChain: destinationChain,
+        event: eventType,
+    };
+
+    const res = await httpPost(url, payload);
+
+    if (res.error) {
+        throw new Error(`Error querying gas amount: ${res.error}`);
+    }
+    return res;
+}
+
+function scaleGasValue(chain, gasValue) {
```

Comment:
Add a docstring to explain why this exists (Hedera uses lower decimal number but EVM interaction needs 18 decimals so need to scale it up)